### PR TITLE
[FLO-1756] Example scripts at the-internet/tables

### PR DESCRIPTION
**Pull request by hong-flood**
This PR includes example scripts below:
```
Until.elementTextDoesNotMatch(selectorOrLocator: NullableLocatable, regex: RegExp)
Until.titleIs(title: string)
Until.urlIs(url: string)

By.js(script: EvaluateFn, ...args: any[])

ElementHandle.findElements(locator: Locator | string)
```

### CODE REVIEW

File: examples/the-internet-more-examples/tables.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+			const table2Id: string = await tables[1].getProperty('id')
+			assert(table1Id === 'table1', 'The id of table 1 must be correct')
+			assert(table2Id === 'table2', 'The id of table 2 must be correct')
+		},
+	)
+
+	step(
+		'Test: Do some sort action with tablesorter',
+		async (browser: Browser): Promise<void> => {
+			const table: Locator = By.id('table2')
```
Comment (HieuLeKMS) [inline examples/the-internet-more-examples/tables.ts:43]: My suggestion is that you should remove type annotation in cases like this - when the variable already know its type from the return type of the method. It's unnecessary and makes the code look more dense

File: examples/the-internet-more-examples/tables.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+			const cellsOfFirstRow: ElementHandle[] = await rows[0].findElements(By.tagName('td'))
+			const lastName: string = await cellsOfFirstRow[0].text()
+			const firstActionEdit: ElementHandle = await rows[0].findElement(By.partialLinkText('edit'))
+			const firstActionDelete: ElementHandle = await rows[0].findElement(
+				By.partialLinkText('delete'),
+			)
+
+			await firstActionEdit.click()
+			await browser.wait(Until.urlIs(`${URL}/tables#edit`))
+			await browser.wait(2)
```
Comment (HieuLeKMS) [inline examples/the-internet-more-examples/tables.ts:57]: Are these `await browser.wait(number-of-second)` only used when you debug the script? If so I think you should remove these.

File: examples/the-internet-more-examples/tables.ts (hunk 1 above)
Comment (hong-flood) [reply examples/the-internet-more-examples/tables.ts:43]: Got it.

File: examples/the-internet-more-examples/tables.ts (hunk 2 above)
Comment (hong-flood) [reply examples/the-internet-more-examples/tables.ts:57]: Yes, it is. I'll remove it.

**Review by HieuLeKMS — APPROVED** [review]

**Pull request merged**